CR 7058808 fc-match core dumps when no fonts are installed. FcFonSort interface returns Null.

--- a/fc-match/fc-match.c	Wed Jun 29 15:00:37 2011
+++ b/fc-match/fc-match.c	Wed Jun 29 15:04:43 2011
@@ -174,6 +174,12 @@
 	FcFontSet	*font_patterns;
 	int	j;
 	font_patterns = FcFontSort (0, pat, all ? FcFalse : FcTrue, 0, &result);
+	
+	if (!font_patterns)
+    	{
+            fprintf (stderr, "Can't find any matching font\n");
+            return 1;
+    	}
 
 	for (j = 0; j < font_patterns->nfont; j++)
 	{
